Gopkg.toml

```diff
@@ -118,3 +118,7 @@
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  branch = "master"
```

Comment:
We shouldn't use `master` branch for deps. Please use a specific commit if there is no tagged version.